### ISSUE TITLE
Vaginal microbiomes of breast cancer survivors treated with aromatase inhibitors with and without vulvovaginal symptoms

### ISSUE THREAD

**Issue by Rahila-me** [waldronlab/BugSigDBcuration#345]
Vaginal microbiomes of breast cancer survivors treated with aromatase inhibitors with and without vulvovaginal symptoms - Prasanchit P, - nature
https://www.nature.com/articles/s41598-024-58118-3

**Comment by SvetlanaUP** [comment]
Noted as a self-assigned curation.

**Comment by Scholarpat** [comment]
Hi @Rahila-me 

Well done on your curation. I've made a few changes:

1. **Study Design**: The control group lacked vulvovaginal symptoms (condition of interest), so I think "case-control" will be better suited than "cross-sectional."

2. **Condition**: Breast cancer survivors treated with aromatase inhibitors were recruited, and participants were then contrasted based on the absence and presence of vulvovaginal symptoms. I've changed this field from "Breast carcinoma" to "Vulvovaginitis," which is the closest term I could find.

I also adjusted the signature description.

This curation is now completed @SvetlanaUP.

Thank you!
